Formula/operator-sdk.rb

```diff
@@ -48,7 +49,14 @@ def install
       assert_match commit_regex, version_output
     end
 
-    output = shell_output("#{bin}/operator-sdk init --domain=example.com --license apache2 --owner BrewTest 2>&1", 1)
-    assert_match "failed to initialize project", output
+    mkdir "test"
+
+    chdir "test" do
```

Comment:
Since tests are run in a temporary directory, I'm not sure this is needed.

Reply:
If you have other suggestions for ways to work around the need for a completely empty directory I am open to implementing whatever you feel would work best. This was just the best way I could think of for a workaround

Reply:
What is the existing file that's there? Because AFAIK that test directory is empty when you run the tests

Reply:
That'll be the hidden `.brew_home` directory.

Reply:
The exact error gotten is:
```
FATA[0000] failed to initialize project: unable to run pre-scaffold tasks of "base.go.kubebuilder.io/v3": target directory is not empty (only go.mod, go.sum, files and directories with the prefix ".", files with the suffix ".md" or capitalized files name are allowed); found existing file "Library" 
```
When injecting a test that runs and checks the output of an `ls -la` command, it showed the following as the output of the `ls -la`:

```
total 4
drwx------ 1 linuxbrew linuxbrew  30 May  3 18:36 .
drwxrwxrwt 1 root      root      138 May  3 18:36 ..
-rw-r--r-- 1 linuxbrew linuxbrew  79 May  3 18:36 .bazelrc
drwxr-xr-x 1 linuxbrew linuxbrew  12 May  3 18:36 Library
```

Reply:
Hidden directories are allowed by operator-sdk but in this case it seems like the `Library` directory is causing an issue.

Reply:
Ah forgot `.brew_home` is there in build environments but not test environments.